deploy/ephemeral.yaml

```diff
@@ -350,6 +355,29 @@ objects:
                   }
                 }
               },
+              "groups": {
+                "type": "nested",
+                "properties": {
+                  "id": {
+                    "type": "keyword"
+                  },
+                  "name": {
+                    "type": "keyword"
+                  }
+                  "account": {
+                    "type": "keyword"
+                  },
+                  "org_id": {
+                    "type": "keyword"
+                  },
+                  "created_on": {
+                    "type": "keyword"
+                  },
+                  "modified_on": {
+                    "type": "keyword"
+                  }
```

Comment:
Should these two be `"type": "date_nanos"`?

Reply:
fixed